ios/swift/AmpliSwiftSampleApp/Shared/Ampli/Ampli.swift

```diff
@@ -0,0 +1,766 @@
+/**
+ * Ampli - A strong typed wrapper for your Analytics
+ *
+ * This file is generated by Amplitude.
+ * To update run 'ampli pull swift-ampli'
+ *
+ * Required dependencies: Amplitude-iOS^8.6.0
+ * Tracking Plan Version: 0
+ * Build: 1.0.0
+ *
+ * [View Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/latest)
+ *
+ * [Full Setup Instructions](https://data.amplitude.com/test-codegen/Test%20Codegen/implementation/swift-ampli)
+ */
+
+import Foundation
+import Amplitude
+
+public typealias MiddlewareExtra = [String: Any]
+
+public enum AmpliEnvironment: Int {
+  case development,production;
+}
+
+let ApiKey: [AmpliEnvironment: String] = [
+  .development:"00aa083ba31d20782808820370c15a71",
+  .production:"af568af728fe7ecab9800979089ad112"
+];
+
+let DefaultOptions = [
```

Comment:
Since this is only used of plan values you could make a default AMPPlan here instead of loose dictionary?

Reply:
Good idea, I already replaced to create a plan object directly.